updater/lib/dependabot/dependency_snapshot.rb

```diff
@@ -37,26 +37,82 @@ def self.create_from_job_definition(job:, job_definition:)
     attr_reader :base_commit_sha
 
     sig { returns(T::Array[Dependabot::DependencyFile]) }
-    attr_reader :dependency_files
+    def all_dependency_files
+      @dependency_files
+    end
+
+    sig { returns(T::Array[Dependabot::DependencyFile]) }
+    def dependency_files
```

Comment:
Minor:

```suggestion
    def dependency_files_for_current_directory
```

For clarity maybe? 🤷

Reply:
Outside of the class it's only used in `update_all_versions` which doesn't have the concept of a current directory. I'd say leave this for now and we'll see how future refactors feel about it 🤷 .